src/paramsUi.cpp

```diff
@@ -562,7 +568,9 @@ class ParamsDialog {
 				// A function key.
 				(VK_F1 <= msg->wParam && msg->wParam <= VK_F12) ||
 				// Anything with both alt and shift.
-				(alt && shift)
+				(alt && shift) ||
+				// Or Space.
+				(msg->wParam == VK_SPACE)
```

Comment:
If I read this correctly, this will cause space to play/stop in edit fields. That will prevent space from being typed into an edit field, which is bad for the filter in particular.

Reply:
> If I read this correctly, this will cause space to play/stop in edit fields. That will prevent space from being typed into an edit field, which is bad for the filter in particular.
> 
Space is working fine in edit fields, have already verified that on Windows and Mac. I don't really know why either, some other internal REAPER handling I guess, but it does work, and being able to hit the usual keystroke to play is a nice little intuition upgrade.

Reply:
So... what happens if you remove this code? This bit of code is only related to edit fields.

Reply:
> what happens if you remove this code?
> 
I'm terrible at reading snippets on here, sorry. Removing what I thought you wanted me to try stops it from building lol.
Do you want it tested without the if statement handling F12/Alt and Shift/Space, or something else?